src/lib/AnnotationModule.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    bdlBoxBlue,
+    bdlGreenLight,
+    bdlWatermelonRed,
+    bdlYellorange,
+    bdlYellow,
+    bdlGrimace,
+} from 'box-ui-elements/es/styles/variables';
+import { ANNOTATION_COLOR_KEY } from './constants';
+import Cache from './Cache';
+
+export const AnnotationColor = {
```

Comment:
Should this move to a constants file? Then we could avoid having to keep importing from `AnnotationsModule` in other files that aren't necessarily related. Maybe we need to have a directory for annotations like
```
src/lib/annotations
src/lib/annotations/AnnotationModule.ts
src/lib/annotations/constants.ts
src/lib/annotations/index.ts
```
Thoughts?

Reply:
I think we should be okay with leaving it as is. The only main usage is in `DocControls`, since it's just this one location, I don't know if we need to have a separate directory for it. Open to other suggestions though.